website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -47,9 +54,22 @@ To cancel a scheduled auto destroy run in Terraform Cloud:
 1. Under **Automatically destroy**, click **Edit** next to your scheduled run's details.
 1. Click **Remove**.
 
-You can schedule [notifications](/terraform/cloud-docs/workspaces/settings/notifications) to alert you 12 and 24 hours before an auto destroy run, and to report auto destroy run results.
+#### Destroy if Inactive
+
+You can configure Terraform Cloud to automatically destroy a workspace's infrastructure after a period of inactivity. Any runs that update Terraform state will push back the scheduled auto destroy time by the configured activity duration designated by the user.
+
+To schedule an auto destroy run after a period of inactivity in Terraform Cloud:
+1. Navigate to the workspace's **Settings** > **Destruction and Deletion** page.
+1. Under **Automatically destroy**, click **Set up auto destroy**.
+1. Click the **Destroy if inactive** toggle.
+1. Enter the desired activity duration.
```

Comment:
Does this populate a suggested default? Or do we have a suggested default? Just curious!

Reply:
There's a select with the default being `Last 1 day` on creation 
<img width="224" alt="image" src="https://github.com/hashicorp/terraform-docs-common/assets/33099255/a325dfac-729f-4cbc-8617-0a6c9b3e24ba">

with multiple presets to choose from
<img width="215" alt="image" src="https://github.com/hashicorp/terraform-docs-common/assets/33099255/a23cdf5e-8f90-4a61-8b0e-c3b370ef419d">

and a more granular `Custom` option
<img width="411" alt="image" src="https://github.com/hashicorp/terraform-docs-common/assets/33099255/0858848d-52f0-4264-99b0-0f168a8b82ee">

Reply:
Oooh thank you so much for the _screenshots_! That really helps! Adding suggestions based on this and sorry for not just going and looking at it myself 😅